README.md

```diff
@@ -270,6 +272,20 @@ export SPHERE_CLIENT_SECRET="your-client-secret"
 #### ElasticIo
 _(Coming soon)_
 
+#### UserAgent
+A synchronous module that builds _user\_agent_ according to the standard specified [here](https://sphere.atlassian.net/wiki/x/7ACjB)
+
+sphere-node-sdk module must be installed in the node_modules because it's required to build the user_agent
+
+```js
+const user_agent = userAgent('sphere-node-utils', '1.0.0')
```

Comment:
I think the reason for that was because of the name already used in the main module, the user can change it to what ever he/she deem fit though